.npmignore

```diff
@@ -0,0 +1,32 @@
+.idea
+
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage
+.grunt
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+node_modules/
+
+temp/
+test/sandbox
+test/.pkgcache/
+.pkgcache/
+test
```

Comment:
We should not be committing `.npmignore` file, it's automatically generated from `.gitignore` by `slt-release` tool.

When `.npmignore` is versioned, then we must ensure it stays in sync with `.gitignore`, which is difficult to do. 

Can you please open a new pull request to remove `.npmignore` from this repository?

Reply:
@bajtos sure. It's auto generated by the releasing tool, I thought it's a missing file. The removing PR is coming.